app/views/comments/_single_comment.html.erb

```diff
@@ -59,8 +59,10 @@
         <% end %>
       <% end %><!-- end caching -->
 
+      <% if logged_in_as_admin? %>
+        <p class="comment ip"><%= ts("IP Address: %{ip_address}", ip_address: single_comment.ip_address.blank? ? "No address recorded" : single_comment.ip_address) %></p>
```

Comment:
Yep, it is indeed the code I gave you, but you should probably try recaching the skins to check that it works instead of just taking my word for it